docs/howto/windows.txt

```diff
@@ -1,14 +1,17 @@
-================================
-How to install Django on Windows
-================================
+==================================================================================
+How to install Django on Windows with Python 3.5, virtualenv and virtualenvwrapper
```

Comment:
I think the existing title is okay and doesn't need to describe all the specifics.